dev-guide/host-port-registry.md

```diff
@@ -88,6 +88,7 @@ Ports are assumed to be used on all nodes in all clusters unless otherwise speci
 | 9103  | ovn-kubernetes node kube-rbac-proxy | sdn || metrics |
 | 9105  | ovn-kubernetes node kube-rbac-proxy-ovn-metrics | sdn | 4.10 | metrics |
 | 9106  | sdn controller kube-rbac-proxy | sdn | 4.10 | control plane only, sdn only |
+| 9107  | ovn-kubernetes node kube-rbac-proxy | sdn | 4.12 | egressip-node-healthcheck-port, node only |
```

Comment:
Should it be: "ovn-kubernetes node" without kube-rbac-proxy?

Reply:
Good question. I was a little hesitant about my copy and paste abilities here.
When using hypershift, there will be a socks-proxy involved in order to get the connection from the leader master node to the worker node, but I think you are correct to question whether that has anything to do with kube-rbac-proxy at all. I will confirm that is not the case and adjust accordingly.